diff --git a/node_modules/@nivo/sankey/dist/nivo-sankey.es.js b/node_modules/@nivo/sankey/dist/nivo-sankey.es.js
index eb9d241..c58974d 100644
--- a/node_modules/@nivo/sankey/dist/nivo-sankey.es.js
+++ b/node_modules/@nivo/sankey/dist/nivo-sankey.es.js
@@ -578,7 +578,7 @@ var SankeyLinksItem = function SankeyLinksItem(_ref2) {
     startColor: link.startColor || link.source.color,
     endColor: link.endColor || link.target.color
   }), React.createElement(animated.path, {
-    fill: enableGradient ? "url(\"#".concat(encodeURI(linkId), "\")") : animatedProps.color,
+    fill: enableGradient ? `url(#${encodeURI(linkId)})` : animatedProps.color,
     d: animatedPath,
     fillOpacity: animatedProps.opacity,
     onMouseEnter: isInteractive ? handleMouseEnter : undefined,
